secant/Features/BackupFlow/RecoveryPhraseDisplayStore.swift

```diff
@@ -15,9 +15,31 @@ enum RecoveryPhraseError: Error {
 }
 
 struct BackupPhraseEnvironment {
-    var mainQueue: AnySchedulerOf<DispatchQueue>
-    var newPhrase: () -> Effect<RecoveryPhrase, RecoveryPhraseError>
-    var pasteboard: UIPasteboard = .general
+    let mainQueue: AnySchedulerOf<DispatchQueue>
+    let newPhrase: () -> Effect<RecoveryPhrase, RecoveryPhraseError>
+    let setPasteboard: (String) -> Void
+    let getPasteboard: () -> String?
+}
+
+extension BackupPhraseEnvironment {
+    private struct DemoPasteboard {
```

Comment:
This can be moved somewhere else later, if we need a testable pasteboard for other features.